scripts/qa_crowbarsetup.sh

```diff
@@ -98,6 +98,8 @@ export want_postgresql=${want_postgresql:-1}
 if iscloudver 6plus ; then
     export CROWBAR_EXPERIMENTAL=true
     export CROWBAR_VERIFY_SSL=false
+    # set timeout to 10min to avoid bsc#988030
+    export CROWBAR_TIMEOUT=600
```

Comment:
@jdsn the old timeout before the change yesterday was client side 1 hour and server side 25 min. where is the 5 min coming from?